src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2981.cs

```diff
@@ -1,6 +1,4 @@
-﻿#if !WINDOWS
-// This test won't work on Windows right now because we can only test desktop, so touch events
-// (like LongPress) don't really work. The test should work manually on a touch screen, though.
+﻿#if TEST_FAILS_ON_CATALYST // TouchAndHold not supported in catalyst
```

Comment:
Right, I am going to implement it in a separate PR.

Reply:
Yes, I have tried `LongPress,`  it also not works on Catalyst, getting a following exception: 
<img width="873" alt="image" src="https://github.com/user-attachments/assets/2efa9436-fdbe-4ab1-8a12-6ba3127998bf">

Reply:
Ok, applied changes to fix it. Triggering the build again.